### ISSUE TITLE
Contents file_info and glob yields error: failed to copy: archive/tar: write too long

### ISSUE THREAD

**Issue by idcmp** [goreleaser/nfpm#441]
Supply some contents using a wildcard, and set a file_info for the src/dst tuple, and nfpm will fail with the above error. Using v2.11.3 with the deb packager on Ubuntu 20, Go 1.16.

**Comment by caarlos0** [comment]
Can you provide a reproducible of that bug?

**Comment by benyanke** [comment]
I'm not working with @idcmp but I encountered this as well.

```yaml
contents:
- src: completion/*
  dst: /etc/bash_completion.d/
  file_info:
    mode: 0755
```

I tried to put a number of completion files in the directory, and then tried to glob it so that I didn't need to declare each one.

nfpm returned the following error when running:

```shell
$ nfpm package --packager deb
using deb packager...
DEPRECATION WARNING: Leaving the 'maintainer' field unset will not be allowed in a future version
cannot write header of completion/kubetools-k9s to data.tar.gz: archive/tar: missed writing 2613 bytes
```

I did indeed not set the maintainer field in my config, so that warning is legitimate, but leaving it in the results, in case it's somehow related.

**Comment by caarlos0** [comment]
should have been fixed by https://github.com/goreleaser/nfpm/pull/482, released in v2.15.1

let me know otherwise.

thanks!
